main.go

```diff
@@ -125,6 +134,27 @@ func applyCommand() *exec.Cmd {
 	)
 }
 
+func assumeRole(roleArn string) {
+    client := sts.New(session.New())
+    duration := time.Hour * 1
+    stsProvider := &stscreds.AssumeRoleProvider{
+        Client: client,
+        Duration: duration,
+        RoleARN: roleArn,
+        RoleSessionName: "drone",
+    }
+
+    value, err := credentials.NewCredentials(stsProvider).Get()
+	if err != nil {
+		fmt.Println("Error assuming role!")
+		fmt.Println(err)
+		os.Exit(1)
+	}
+    os.Setenv("AWS_ACCESS_KEY_ID",value.AccessKeyID)
```

Comment:
Since this is running in the terraform plugin container itself, the environment variables should be available to the rest of the terraform run, correct? I did some tests and it does appear to be working properly.